examples/weather/graphcast/conf/config.yaml

```diff
@@ -32,18 +35,18 @@ checkpoint_encoder_finetune: True
 checkpoint_processor_finetune: True
 checkpoint_decoder_finetune: True
 concat_trick: True
-cugraphops_encoder: False
-cugraphops_processor: False
-cugraphops_decoder: False
-recompute_activation: False
+cugraphops_encoder: True
+cugraphops_processor: True
+cugraphops_decoder: True
+recompute_activation: True
 wb_mode: "disabled"
 synthetic_dataset: false
 dataset_path: "/data"
-static_dataset_path: "datasets/static"
-latlon_res: (721, 1440)
-num_samples_per_year_train: 1448
+static_dataset_path: null
+latlon_res: [721, 1440]  
+num_samples_per_year_train: 1408
 num_workers: 0  # 8
-num_channels: 3  # 34
+num_channels: 474
```

Comment:
since we are revisiting this: I feel like we have been a bit inconsistent when it comes to "num_channels". In GraphCast, we technically have `num_channels = num_data_channels * (1 + history_len) + static_channels`. (and the number of static channels technically also depends on the length of history)

For now, the model neither has been fully agnostic to how the input is actually prepared nor fully aware of it. We included the option for a "static" dataset in GraphCast itself but would have assumed the history to be taken care of outside of the model. 

Would it make sense to move the static dataset outside of GraphCast, too, and then just have "num_channels", i.e. have an agnostic model when it comes to input preparation? 

Or should be add a "prepare_input" method which applies the history and the static dataset and let the model compute "num_channels" based on "num_data_channels", "num_static_channels", and "history_len"?

Reply:
I vote in favor of moving the static dataset outside of GraphCast. If anything, I could imagine a wrapper class that does ERA5 specific utilities like appending types of static data.

Reply:
Done, please take a look.